ledger/tracker.go

```diff
@@ -346,9 +363,9 @@ func (tr *trackerRegistry) loadFromDisk(l ledgerForTracker) error {
 		}
 	}
 
-	err := tr.replay(l)
+	err = tr.replay(l)
 	if err != nil {
-		err = fmt.Errorf("initializeTrackerCaches failed : %w", err)
+		err = fmt.Errorf("trackers replay failed : %w", err)
 	}
 	return err
```

Comment:
I'd find this easier to follow if the err was immediately returned, else return `nil` explicitly in the good case.
```
	err = tr.replay(l)
	if err != nil {
		return fmt.Errorf("trackers replay failed : %w", err)
	}
    return nil
```